README.md

```diff
@@ -366,6 +360,8 @@ You may also [browse the Supermarket for shared Compliance Profiles](https://sup
 
 ## Kudos
 
+Chef InSpec was originally created by Christoph Hartmann (@chris-rock) and Dominik Richter (@arlimus).
```

Comment:
```suggestion
Chef InSpec was originally created by Christoph Hartmann and Dominik Richter.
```

Those are Slack handles, but Chris and Dominik have moved on to different companies.

Reply:
They are also their github IDs.